src/Http/Controllers/QrController.php

```diff
@@ -27,7 +27,12 @@ public function get(Request $request)
             throw new PlatformException(__('enjin-platform::error.qr.extension_not_installed'), 501);
         }
 
-        $qrCode = QrCode::format($format)->size($size)->generate($data);
+        $qrCode = QrCode::format($format)
+            ->size($size)
+            ->errorCorrection('Q')
+            ->merge(config('enjin-platform.qr.image'), config('enjin-platform.qr.image_size'), true)
```

Comment:
Might need to bypass this if a QR Image hasn't been set in the .env, or at least test it works without an image :)